### DISCUSSION TITLE
Welcome to CREXX Discussions!

### DISCUSSION THREAD

**Discussion by adesutherland** [adesutherland/CREXX#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rvjansen** [comment]
Hi there! I am René Jansen and I am president of the Rexx Language Association. Using Rexx for the entirety of my professional life, and not willing to be one day without it. I started out as a systems programmer on MVS/XA, on the big iron, and met Rexx not long after that. I am excited that we are trying a from-scratch Classic Rexx interpreter and compiler, with the aim of serving the current platforms and peek into the future of Rexx. Having it run on VM/370 as it runs on its siblings, but also having a nice 64-bit Rexx compiler in our future are important goals of the projects. Current interests are NetRexx, Java as the big equaliser, and new trends like LLVM and WASM. Also, strong love for small platforms like aarch64 (ARM-64); day to day job still on z/OS.

**Comment by BobBolch** [comment]
Hello. I am Bob Bolch. I spent my career as a System Programmer, Software
Developer, and Software Architect.
I started with IBM VM/370 in 1974 and retired in 2015. I started using REXX
when it was introduced in the
IBM VM/SP product in 1983. Most of the software I developed used a
combination of REXX and assembler.

I started working with Adrian, almost a year ago to upgrade the version of
BREXX which had been ported to
VM/370 running on the Hercules mainframe emulator. We have added
capabilities from later VM systems
back into VM/370 to create a more robust and full featured REXX capability
there.

I agree with René that I have to have REXX in my computing environment!

**Comment by mgrossmann** [comment]
Hi...

I am Mike Großmann. I spent the first 18 years of my life as a software developer and software architect in the free, distributed world. During this time I was programming mainly in Java and other JVM languages. Then 3 years back I came to the mainframe. In the beginning, it was about making Java interoperate with existing assembler modules there. So I quickly came into contact with assembler, C and also REXX. When I installed MVS3.8j TK4- for the first time at home, it was a shock for me. No ISPF and also no REXX. On my naive questions to Jürgen Winkelmann he told me that I could take the rudimentary ported BREXX and add what I was missing. In the meantime, two years of development have resulted from this. So this is my background. I am interested in this project because I have learned to love REXX and I am always looking for exciting topics.

/MIG
